docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
       RAILS_ENV: development
       DATABASE_URL: postgresql://devto:devto@db:5432/PracticalDeveloper_development
       YARN_INTEGRITY_ENABLED: "false"
-      ELASTICSEARCH_URL: http://elasticsearch:9200
+      ELASTICSEARCH_URL: elasticsearch
```

Comment:
Does this actually work for you? The URL doesn't specify the protocol or port, which seems odd.

Reply:
To clarify: `elasticearch` only replace the IP/domain part of the URL, but we still need to specify the port etc.

Reply:
It worked, but I didn't know that we need to explicitly specify the port and protocol :sweat_smile: